package_manifest.yaml

```diff
@@ -643,6 +643,7 @@ foreman_plugin_packages_tier2:
     rubygem-foreman_wreckingball: {}
     rubygem-llhttp: {}
     rubygem-puppetdb_foreman: {}
+    rubygem-smart_proxy_request_forwarder: {}
```

Comment:
I think `foreman_proxy_plugin_packages` is a better group for it.

Reply:
Moved it there.